### PR TITLE
[REVIEW] Cleanup for better automation

### PR DESCRIPTION
**Pull request by beckernick**
This PR:
- Makes several small updates to allow for cleaner automation

### CODE REVIEW

File: tpcx_bb/xbb_tools/utils.py [hunk 1]
```diff
@@ -772,15 +772,15 @@ def build_benchmark_googlesheet_payload(config):
     payload = OrderedDict(
         {
             "Query Number": QUERY_NUM,
-            "Protocol": data.get("protocol"),
+            "Protocol": "UCX" if data.get("nvlink") == True else "TCP",
```
Comment (VibhuJawa) [inline tpcx_bb/xbb_tools/utils.py:775]: Should this be if it's either IB or NVLINK, or is the assumption here that we don't use IB without NVLINK (Which is true but maybe add a comment to clear this up ?)

File: tpcx_bb/xbb_tools/utils.py [hunk 2]
```diff
@@ -790,12 +790,11 @@ def build_benchmark_googlesheet_payload(config):
             if read_graph_creation_time
             else "NA",
             "Machine Setup": data.get("hostname"),
+            "Number of GPUs": data.get("num_workers"),
             "Data Location": data.get("data_dir"),
-            "Repartition_small_table": data.get("repartition_small_table"),
-            "Result verified": data.get("result_verified"),
             "Current Time": current_time,
-            "Device Memory Limit": data.get("device_memory_limit"),
```
Comment (VibhuJawa) [inline tpcx_bb/xbb_tools/utils.py:797]: Did we remove this cause we don't set our cluster up via a cli anymore ?

**Review by VibhuJawa — COMMENTED** [review]
Some clarification questions looks good to me otherwise.

File: tpcx_bb/xbb_tools/utils.py (hunk 1 above)
Comment (beckernick) [reply tpcx_bb/xbb_tools/utils.py:775]: Assumption is that if NVLink is enabled we must be using UCX. We may be using IB, but may not be. We have another dictionary key to capture IB, though it's not doing anything at the moment.

File: tpcx_bb/xbb_tools/utils.py (hunk 2 above)
Comment (beckernick) [reply tpcx_bb/xbb_tools/utils.py:797]: Yeah. We don't really have a clean way of capturing this at the moment, so it's not actually doing anything as is. There are several others like this (such as Infiniband).

I think we can solve all of these in a follow up, and keep the status quo

Comment (VibhuJawa) [reply tpcx_bb/xbb_tools/utils.py:797]: Agreed

File: tpcx_bb/xbb_tools/utils.py (hunk 1 above)
Comment (VibhuJawa) [reply tpcx_bb/xbb_tools/utils.py:775]: Gotcha, That makes sense.

**Review by VibhuJawa — APPROVED** [review]
LGTM

**Pull request merged**